airflow/config_templates/config.yml

```diff
@@ -1028,6 +1028,27 @@ secrets:
       type: string
       example: ~
       default: ""
+    use_cache:
+      description: |
+        Enables local caching of Variables, when parsing DAGs only.
```

Comment:
I think we will also need a note in "best practices" and "top-level-code" especially where we explicitly mention "don't use variables in top-levle"